bibliopixel/animation/strip.py

```diff
@@ -5,14 +5,11 @@
 class BaseStripAnim(Animation):
     LAYOUT_CLASS = Strip
     LAYOUT_ARGS = 'num',
+    FAIL_ON_WRONG_LAYOUT = False
```

Comment:
Please set this in BaseStripAnim. I don't like magic values that only appear when needed and for which their default is in some random line of code. I understand this is entirely "valid" but it makes me nauseous and it makes the code VERY hard to read and understand.

Reply:
Done!